masschallenge/django-accelerator
# [AC-9532]: Remove judge_interest, mentor_interest, and speaker_interest fields from CoreProfile model

[AC-9532](https://masschallenge.atlassian.net/browse/AC-9532)

How to test:
- Ensure the definition of done on [AC-9532](https://masschallenge.atlassian.net/browse/AC-9532) are met
